libs/utils/utils.js

```diff
@@ -1295,3 +1297,34 @@ export function loadLana(options = {}) {
 }
 
 export const reloadPage = () => window.location.reload();
+
+export const userCanPublishPage = async (detail, isBulk = true) => {
+  if (!detail) return false;
+  const { live, profile, webPath } = detail;
+  let canPublish = isBulk ? live?.permissions?.includes('write') : true;
+  let message = 'Publishing is currently disabled for this page';
+  const config = await getCustomConfig('/.milo/publish-permissions-config.json');
+  if (config) {
```

Comment:
I feel like you could further leverage optional chaining here and at least remove one level of `if` statement... maybe something like this?

```js
const item = config?.urls?.data?.find(({ url }) => (
  url.endsWith('**') ? webPath.includes(url.slice(0, -2)) : url === webPath
));
```
if you put that directly below `config`'s definition, `item`'s definition should safely short-circuit with `config?.` in the event that `config` is falsy, in which case `item` would also be falsy, and you could then accurately use the following `if (item)` statement.

Reply:
Thanks @elan-tbx I will take that recommendation!